routes/api.php

```diff
@@ -25,4 +25,5 @@
     $router->get('/invite-key/{inviteKeyId}', [UserController::class, 'getInviteKey']);
     $router->get('/game/{gameId}/users', [GameController::class, 'getUsersInGame']);
```

Comment:
Klopt het dat de web routes `/games/{id}` zijn en de API routes `/game/{id}` (games/game)?

Reply:
Klopt. Volgende sprint wordt alles dus REST gemaakt. Voor nu zijn de API routes nog in enkelvoud (maar hier komt dus een heel ander stukje refactoring bij kijken nog).

Reply:
> Als ik een spel start en op de knoppen pauzeren of beëindigen klik komt er een error message:
> 
> * The duration field is required.
> * The interval field is required.

fixed